IFC4x3/Properties/c/CoveragePercentage_0zErkL8B54XBTK$nUEKqm5/Documentation.md

```diff
@@ -0,0 +1 @@
+Related to the MappedZone, e.g.% of the tunnel face where a certian ground types was documented.
```

Comment:
`MappedZone` is formatted differently than in the `AreaID` description.

Reply:
Suggestion: Rewrite to "Coverage in percentage of the mapped feature in relation to the mapped zone, e.g. tunnel face, to which it belongs."
Ping @SergejMuhic and @j-weil.